src/services/authService.js

```diff
@@ -107,10 +111,10 @@ export default class AuthService {
 
     const result = await User.create(userDetails);
     const { dataValues: user } = result;
-    const payload = Helper.pickFields(user, ['id', 'is_admin']);
+    const payload = Helper.pickFields(user, ['id', 'roleId']);
     const token = await Helper.genToken(payload);
 
-    return { token, ...Helper.omitFields(user, ['password']) };
+    return { token, ...Helper.omitFields(user, ['password', 'roleid', 'createdat', 'updatedat']) };
```

Comment:
Why  'createdat', 'updatedat' and `roleid`

Reply:
@Annmary12  i have been able to take care of the roleid, but anytime an insert query is been made on the database sequelize returns a new createdAt and updatedAt all in small case

Reply:
I noticed that too. when you are inserting a new data, it returns two `createdAt` and `updatedAt`. I also noticed that in your team's PR. This will not hold your PR from not merging but I will like the team to figure out why this happens.

Reply:
ok. thanks